cypress/e2e/euronews/assertionData.js

```diff
@@ -0,0 +1,8 @@
+export const assertions = {
```

Comment:
General comment:
Tests look much better and cleaner.
There are still a couple of hardcodes and minor parts to improve.